src/SFML/Graphics/Shape.cpp

```diff
@@ -275,14 +255,14 @@ void Shape::updateOutline()
         Vector2f p2 = m_vertices[index + 1].position;
 
         // Compute their normal
-        Vector2f n1 = computeNormal(p0, p1);
-        Vector2f n2 = computeNormal(p1, p2);
+        Vector2f n1 = (p0 - p1).perpendicular();
+        Vector2f n2 = (p1 - p2).perpendicular();
```

Comment:
Are you sure we want `perpendicular` here and not `normalized`?

Reply:
I believe so. I double checked the implementation of `computeNormal` on paper and saw that it was returning a vector perpendicular to the vector defined by the two points passed as arguments. "Normal" in this context is referring to this:

https://en.wikipedia.org/wiki/Normal_(geometry)

Reply:
But the original code was also computing the length and dividing the vector by such length. Was that unnecessary?

Reply:
Good point. Idk how necessary the normalization is but I'll add that back since I can't confirm whether or not we need it.

Reply:
Fixed now. If you'd rather I not add a local lambda and instead keep that as a free function then I can do that to minimize the diff.